mcstatus/address.py

```diff
@@ -229,3 +232,7 @@ async def async_minecraft_srv_address_lookup(
         port = default_port
 
     return Address(host, port)
+
+
+# some hacks for the documentation
+del _AddressBase.__new__.__doc__
```

Comment:
Why is this necessary? Can't such an ignore be done directly in `docs/`? I'd rather not clutter the actual code with it.

Reply:
Removed [here](https://github.com/py-mine/mcstatus/compare/bb194c20f1882cd4832076a1360070ec9bffce64..405901f3971b240e4b3b576c409cadbfe0b121cf) and added to `conf.py` [here](https://github.com/py-mine/mcstatus/compare/7ac401423b7425a434d8f428cde745e6745533cd..cfc4b3e5fa9254fd6e6cd7294f7be796a729d04d).